instrumentation/kamon-armeria/src/test/resources/application.conf

```diff
@@ -0,0 +1,4 @@
+kamon.instrumentation.armeria.server.tracing.operations.mappings = {
```

Comment:
These settings should not be necessary after changing the way we generate server span names